crates/arroyo-types/src/lib.rs

```diff
@@ -104,6 +104,7 @@ pub const GRPC_PORT_ENV: &str = "GRPC_PORT";
 pub const HTTP_PORT_ENV: &str = "HTTP_PORT";
 pub const COMPILER_PORT_ENV: &str = "COMPILER_PORT";
 
+pub const UPDATE_AGGREGATE_FLUSH_MICROS_ENV: &str = "UPDATE_AGGREGATE_FLUSH_MICROS";
```

Comment:
I think our other time configs are in milliseconds (like BATCH_LINGER_MS), and I think it's also a bit friendlier to someone configuring it as a reasonable number of micros is a very large number.

Reply:
Done